### ISSUE TITLE
Numpy log is flodding CI output

### ISSUE THREAD

**Issue by larroy** [apache/incubator-mxnet#16413]
Please reduce logging since this is going to fill up the logs.

```
Nu
mPy-shape semantics has been activated in your code. This is required for creating and manipulating scalar and zero-size tensors, which were not supported in MXNet before, as in the official
NumPy library. Please DO NOT manually deactivate this semantics while using `mxnet.numpy` and `mxnet.numpy_extension` modules.
 semantics while using `mxnet.numpy` and `mxnet.numpy_extension` modules.
and `mxnet.numpy_extension` modules.
NumPy-shape sem
```

**Comment by larroy** [comment]
@mxnet-label-bot add [Numpy]

**Comment by haojin2** [comment]
Solved via #16849, closing now.
